garden/install_ubuntu.md

```diff
@@ -15,14 +15,14 @@ First install some necessary tools:
 
 ```bash
 sudo apt-get update
-sudo apt-get install lsb-release wget gnupg
+sudo apt-get install lsb-release gnupg
```

Comment:
Does `curl` come installed by default? Some of the instructions (e.g. install_ubuntu_src.md) has `curl` here.

Reply:
It comes by default in docker images and most of the Ubuntu flavors. Anyway, given the number of derivatives based on Ubuntu it is worth to make it explicit:  [2bf4d25](https://github.com/gazebosim/docs/pull/451/commits/2bf4d250b4f25b0a4e24882fc1f310f6bb279e60)